uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/apis/leave-request.api.js

```diff
@@ -126,22 +138,11 @@ define([
          */
         create: function (params) {
           $log.debug('LeaveRequestAPI.create', params);
-          var deferred = $q.defer();
-
-          if (params) {
```

Comment:
There a couple of reasons there were removed:
1) Consistency with the back-end. Once the validation on the back-end is changed it also needs to be changed on the front end. There were already issues with that in the past.
2) These messages are not UX friendly.
3) This is not the case when additional FE validation helps since the Leave Request creation process is not that frequent or is a one step process.